### PR TITLE
Use the eurucamp styleguide and adapt the styles from there [WIP]

### PR DESCRIPTION
**Pull request by moonglum**

### CODE REVIEW

**Comment by moonglum** [comment]
screen shot 2015-03-12 at 23 18 55

**Comment by moonglum** [comment]
I'm not sure if there's anything more to do for the design... There's not a lot of design to speak of in this app :wink:

**Comment by eljojo** [comment]
it might be some privacy blocking thing, but the twitter and facebook buttons are not in the screenshot.
screen shot 2015-03-12 at 23 46 36

also, what's up with the eurucamp flying flag?
screen shot 2015-03-12 at 23 47 17

File: app/assets/stylesheets/framework_and_overrides.css.scss [hunk 1]
```diff
@@ -4,14 +4,26 @@
 $body-font-family:   'sonarnormal', sans-serif;
 $header-font-family: 'sonarnormal', sans-serif;
 
-$eurucampRed: #f6271b;
-$eurucampYellow: #fafb47;
+$eurucampRed: #E500FF;
```
Comment (bascht) [inline app/assets/stylesheets/framework_and_overrides.css.scss]: screenshot from 2015-03-14 19 29 24
That's a pretty liberal interpretation of »red« :smiley_cat:

**Comment by moonglum** [comment]
Thanks @eljojo :+1: 

Yes, that is entirely correct, @bascht :tongue:

**Comment by maser** [comment]
Can this be merged?

**Comment by moonglum** [comment]
Nope, as discussed in the channel, @nerdbabe will create a background image and I will create an unified look for all conferences.

**Pull request closed without merging**